_pages/develop/map.md

```diff
@@ -0,0 +1,682 @@
+---
+title: Developing with px-map
+moduleName: view-develop-map
+pathToRoot: ../../
+layout: default
+---
+
+<style>
+  .gif {
+    width: 100%;
+    margin-top: .25rem;
+  }
+</style>
+
+# Overview
+Px-map is a lightweight framework for building interactive maps with web
+components and Polymer. The framework includes a base mapping component (`<px-map>`)
+that can be used with many different subcomponents to solve common mapping UI
+problems, like plotting geographic data.
+
+The current major release of px-map uses the open source [Leaflet](http://leafletjs.com/)
+library to support displaying a map with base tiles and geospatial overlays.
+Future iterations of the component may offer the ability to use a different
+library in place of Leaflet, while keeping the same basic API.
+
+Out of the box, `px-map` includes the basic things needed to draw and plot data
+on an interactive map. It is also extensible — development teams are encouraged
+to build their own subcomponents that add features needed for their application.
+We currently do not support visualizations, but any open source Leaflet plugin
```

Comment:
If we "do not support visualizations," why do we list visualization layers as a heading further down?